lib/travis/cli/repo_command.rb

```diff
@@ -28,6 +28,7 @@ def repository
         repo(slug)
       rescue Travis::Client::NotFound
         error "repository not known to #{api_endpoint}: #{color(slug, :important)}"
+        error "you can override the auto-detected repo slug with the -r option"
```

Comment:
I don't think the execution reaches here, since `error` call includes `exit`. If you want to alert the user, you'd need to include the text in the `error` line above.